extensions/amp-story-auto-analytics/0.1/auto-analytics-configs.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @param {string} gtagId
+ * @return {!Object}
+ */
+export const buildGtagConfig = (gtagId) => ({
+  'vars': {
+    'gtag_id': gtagId,
+    'config': {
+      [gtagId]: {
+        'groups': 'default',
+      },
+    },
+  },
+  'triggers': {
+    'storyPageCount': {
+      'on': 'story-page-visible',
```

Comment:
Wondering if we should add by default the `"repeat": false` option on the `storySpec` property for this trigger. 
This ensures that the trigger doesn't get fired twice when a user goes back and forth between pages in the story. At the end, it provides a clearer view in their metrics (e.g. slicing by # of users that arrived to page 5 by counting number of `story-page-visible` triggers in their analytics).

This may be a question for @raovs, should we add this option by default?

Reply:
It does make sense, I'm using the config from the DD but will wait for @raovs.

Reply:
Can we send this PR and send a followup if needed? We will likely want to add `repeat: false`, but let's not block this PR. The dashboards template were built without this extra flag, so things should still work in the meantime.

Reply:
We agreed that it's a good idea to add the repeat, currently working on the tests to send the commit

Reply:
It's not up to us to make this decision though, let's wait until we hear back from the people using this extension.

Reply:
Let's move this discussion to offline so we can merge this and update on a future PR